docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -271,11 +273,13 @@ table may be of use:
 
 | Silverstripe CMS Version | PHP Version | More information |
 | -------------------- | ----------- | ---------------- |
-| 3.0 - 3.5            | 5.3 - 5.6   | [requirements docs](https://docs.silverstripe.org/en/3.4/getting_started/server_requirements/)
+| 3.0 - 3.5            | 5.3 - 5.6   | |
 | 3.6                  | 5.3 - 7.1   | |
 | 3.7                  | 5.3 - 7.4   | [changelog](https://docs.silverstripe.org/en/3/changelogs/3.7.4/) |
 | 4.0 - 4.4            | 5.6+        | |
-| 4.5+                 | 7.1+        | [blog post](https://www.silverstripe.org/blog/our-plan-for-ending-php-5-6-support-in-silverstripe-4/) |
+| 4.5 - 4.9            | 7.1+        | [blog post](https://www.silverstripe.org/blog/our-plan-for-ending-php-5-6-support-in-silverstripe-4/) |
+| 4.10                 | 7.3+        | [changelog](/Changelogs/4.10.0#phpeol/) |
+| 4.11 +               | 7.4+        | [changelog](/Changelogs/4.11.0#phpeol) |
```

Comment:
Maybe we want to be explicit here and say those ones are unreleased.

I didn't say which release which release will drop PHP7, because we don't know what number that will be.

Reply:
I think it's fine as is